lib/manager/components/ProjectViewer.js

```diff
@@ -295,3 +307,18 @@ class ProjectSummaryPanel extends Component<{feedSources: Array<Feed>, project:
     )
   }
 }
+
+const ExplanatoryPanel = ({ project }) => {
+  // If user has more than 3 labels, hide the feed source instruction
+  if (project.labels.length <= 3) {
```

Comment:
(non-blocking) I'd be in favor of leaving the condition outside of this component.

Reply:
I originally had this, but this increases the complexity of the main render method beyond what eslint allows